runtime/lib/component/light.js

```diff
@@ -179,10 +182,13 @@ Light.prototype.lightMethod = function (name, args) {
  * @param {boolean} dndMode true if opened
  */
 Light.prototype.setDNDMode = function (dndMode) {
+  var caps = new Caps()
   if (dndMode) {
-    this.lightMethod('setGlobalAlphaFactor', [DND_MODE_ALPHA_FACTOR])
+    caps.writeDouble(DND_MODE_ALPHA_FACTOR)
```

Comment:
#203 adds the method(RPC), I think the `lightMethod` would be rewritten in that way, so a TODO comment is required.

Reply:
AFAIK there is no persist in RPC mechanism. Yet persist message is a requirement here.

Reply:
Yeah, all the dbus method should be refacted to flora.

> #203 adds the method(RPC), I think the `lightMethod` would be rewritten in that way, so a TODO comment is required.

Yeah, all the dbus method should be refacted to flora.